astropy/table/pprint.py

```diff
@@ -185,7 +185,7 @@ def _get_pprint_size(max_lines=None, max_width=None):
 
 
     def _pformat_col(self, col, max_lines=None, show_name=True, show_unit=None,
-                     show_dtype=False, show_length=None, html=False):
+                     show_dtype=False, show_length=None, html=False, align='right'):
```

Comment:
I think this is a leftover from an earlier version since it doesn't appear that `align` is used anywhere in this function.